aten/src/ATen/native/vulkan/VulkanOps.cpp

```diff
@@ -607,7 +607,7 @@ void mul(VulkanTensor& output, const VulkanTensor& input, const float s) {
   const auto sizes = input.sizes();
 
   const auto C = std::accumulate(
-      sizes.cbegin(), sizes.cend() - 2, 1, std::multiplies<int64_t>());
+      sizes.cbegin(), sizes.cend() - 2, (int64_t)1, std::multiplies<int64_t>());
```

Comment:
`const int64_t C` and then decltype type later or C++ style cast instead of C-style cast